adobe/spectrum-css
# feat(tokens): use @adobe/spectrum-tokens@12.27.0 + apply CJK typography changes

## Description

Takes in the `12.27.0` [release](https://github.com/adobe/spectrum-tokens/releases/tag/%40adobe%2Fspectrum-tokens%4012.27.0) of `@adobe/spectrum-tokens` which includes additions and changes for CJK typography. While there are 27 additions or changes in that release, the downstream scope of changes is much more broad, as these changes will impact any consumer using Typography styles for CJK locales, the Illustrated Message component for CJK locales, and the Drop Zone component for CJK locales.

**Affected components:**
* Typography
* Illustrated message (via tokens-only)
* Drop zone (uses Illustrated message) - (via tokens-only)

**Visual testing**
Unfortunately, we don't have visual tests that cover these changes, so Chromatic isn't detecting any changes.

## How and where has this been tested?

Please tag yourself on the tests you've marked complete to confirm the tests have been run by someone other than the author.

### Validation steps
1. Check that the [deployed, localized Typography story](https://64762974a45b8bc5ca1705a2-wrdatabppv.chromatic.com/?path=/story/components-typography--default&args=size:xl&globals=lang:zh) renders with a _smaller_ font (now `18px`) compared to [production](https://opensource.adobe.com/spectrum-css/preview/index.html?path=/story/components-typography--default&args=size:xl&globals=lang:zh) which uses `--spectrum-body-font-size` and renders at `20px`.
2. Given the links in step 1, generally compare the two URLs: everything in the deploy URL should look smaller and have less visual space compared to the production URL.
3. Compare the [deploy URL for Illustrated message](https://64762974a45b8bc5ca1705a2-wrdatabppv.chromatic.com/?path=/story/components-illustrated-message--default&globals=lang:zh) vs. the [production URL](https://opensource.adobe.com/spectrum-css/preview/index.html?path=/story/components-illustrated-message--default&globals=lang:zh) - and observe that the deploy URL's typography should be smaller and have less space compared to the production URL (similar to what you saw on the Typography component pages).
4. Compare the [deploy URL for Drop zone](https://64762974a45b8bc5ca1705a2-wrdatabppv.chromatic.com/?path=/story/components-drop-zone--default&globals=lang:zh) vs. the [production URL](https://opensource.adobe.com/spectrum-css/preview/index.html?path=/story/components-drop-zone--default&globals=lang:zh) - and observe that the deploy URL's typography should be smaller and have less space compared to the production URL (similar to what you saw on the Typography component pages).



### Regression testing

Validate:

1. The documentation pages for at least two other components are still loading, including:


5. If components have been modified, VRTs have been run on this branch:


## Screenshots

BEFORE (current production, XL size):
<img width="1378" alt="Screenshot 2024-09-27 at 11 13 32 AM" src="https://github.com/user-attachments/assets/216658ac-8f12-450d-a070-ecec77fafe47">

AFTER (deploy URL, XL size):
<img width="1383" alt="Screenshot 2024-09-27 at 11 13 23 AM" src="https://github.com/user-attachments/assets/65195166-5f2a-462f-a8e4-0ae806dd9373">

## To-do list
